setup.py

```diff
@@ -36,6 +36,7 @@
     },
     include_package_data=True,
     install_requires=[
+        "biopython",
```

Comment:
I'd *like* to pin this to a recent version, but InSilicoSeq [currently requires biopython ≤ 1.78](https://github.com/bioconda/bioconda-recipes/blob/951e5e70a85089d3c05a05e37832ebdc1af9f53d/recipes/insilicoseq/meta.yaml).